examples/micronaut-client/src/test/groovy/micronaut/client/utils/KubernetesSpecification.groovy

```diff
@@ -0,0 +1,218 @@
+/*
+ * Copyright 2017-2020 original authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package micronaut.client.utils
```

Comment:
is there a reason why we use a package not starting with `io.micronaut` but `micronaut` instead?